test/util/usa.test.js

```diff
@@ -1,11 +1,11 @@
 /* eslint no-undef: 0 */
 
-import lookup, { states } from '../../src/util/usa'
+import lookup, { data as states } from '../../src/util/usa'
 
 describe('usa utility', () => {
   it('should cover all 50 states and DC', () => {
     const stateCount = Object.keys(states).length
-    expect(stateCount).toEqual(51)
+    expect(stateCount).toEqual(52)
```

Comment:
I don't know about this test, I'd say let's just remove it since its really brittle and doesn't actually assure us of that much.